packages/api/models/Error.ts

```diff
@@ -3,12 +3,12 @@ export enum ErrorCode {
   OTHER = "OTHER",
 }
 
-export interface Error {
+export interface ErrorModel {
   code: ErrorCode;
   message: string;
 }
 
-export interface ValidationError extends Error {
+export interface ValidationErrorModel extends ErrorModel {
   code: ErrorCode.INVALID_INPUT;
   error_map: Record<string, unknown>;
 }
```

Comment:
Why do we need to change the name of this to include Model? It shouldn't conflict with an `orm/Entity` because there is no entity for Error.

Reply:
I agree and thought the same. However, I wasn't entirely sure were to draw the line, so I included it since I changed most of the others.